debug_toolbar/settings.py

```diff
@@ -76,6 +76,14 @@ def get_panels():
         PANELS = list(settings.DEBUG_TOOLBAR_PANELS)
     except AttributeError:
         PANELS = PANELS_DEFAULTS
+
+    logging_panel = "debug_toolbar.panels.logging.LoggingPanel"
+    if logging_panel in PANELS:
+        PANELS = [panel for panel in PANELS if panel != logging_panel]
+        warnings.warn(
+            f"Please remove {logging_panel} from your DEBUG_TOOLBAR_PANELS setting.",
+            DeprecationWarning,
+        )
```

Comment:
@tim-schilling Have you seen this block? I have to admit I haven't tested it properly but shouldn't this be sufficient? 

(It wouldn't work if people imported the panel themselves but that wasn't documented anyway.)